.gitlab-ci.yml

```diff
@@ -15,6 +15,7 @@ test-run27:
   before_script:
     - pip install --upgrade pip
     - pip install -r requirements.txt
+    - pip install -r requirements-test.txt
```

Comment:
`requirements.txt` is for testing, so not need to create another `requirements-test.txt`, just add to the regular `requirements.txt`

Reply:
@mrtizmoatwork It's your call. I don't have strong opinions about this.

Reply:
What @sbnetops is saying makes sense to me. Put all of the dependencies in `requirements.txt`, but make sure there are no "testing" deps in `setup.py`'s `install_requires` section.